src/Components/SideBar/SideBar.jsx

```diff
@@ -72,9 +72,9 @@ const SideBar = () => {
       <div className={SideBarStyles.ArticleSection}>
         <div className={SideBarStyles.FilterSection}>
           <ButtonMenu
-            buttonLabel="Type"
+            buttonLabel="Content Type"
             optionsTitle="Filter by Content Type"
-            options={["Race-related incident", "Opinion", "Good deed"]}
+            options={["Incident Report", "Empowering Stories", "Acts of Allyship"]}
```

Comment:
Does changing the values here for options actually allow it to filter? 

Just looking at the [Airtable](https://airtable.com/tblhquAoHSaDKDnKh/viwlfyCtOKawKNrUh?blocks=bipaNlSeLZIeB6tl3), the types we have are "General News", "Empowering Stories", "Acts of Allyship" and "Incident".

Reply:
Also, @SDC-LZC do we also show "Resources", or "Opinions"?

Reply:
> Does changing the values here for options actually allow it to filter?
> 
> Just looking at the [Airtable](https://airtable.com/tblhquAoHSaDKDnKh/viwlfyCtOKawKNrUh?blocks=bipaNlSeLZIeB6tl3), the types we have are "General News", "Empowering Stories", "Acts of Allyship" and "Incident".

My bad. The filter doesn't work. You're right that the Airtable types don't match the front-end.

Reply:
Hey guys -- I think we should show the following (in this order):

- Incident
- Act of Allyship
- Stories of Empowerment
- Resource
- General News

Sorry it was a poorly written ticket.  I can change the Airtable categories to be aligned to the categories written above.

Thank you guys!